omod/src/main/java/org/openmrs/module/locationbasedaccess/web/controller/PatientsMigrationController.java

```diff
@@ -0,0 +1,70 @@
+package org.openmrs.module.locationbasedaccess.web.controller;
+
+import java.util.Date;
+import java.util.Iterator;
+import java.util.List;
+import java.util.Map;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.commons.lang3.StringUtils;
+import org.openmrs.Patient;
+import org.openmrs.Person;
+import org.openmrs.PersonAttribute;
+import org.openmrs.PersonAttributeType;
+import org.openmrs.api.PatientService;
+import org.openmrs.api.PersonService;
+import org.openmrs.api.context.Context;
+import org.openmrs.module.locationbasedaccess.LocationBasedAccessConstants;
+import org.openmrs.module.webservices.rest.SimpleObject;
+import org.openmrs.module.webservices.rest.web.RestConstants;
+import org.openmrs.module.webservices.rest.web.RestUtil;
+import org.openmrs.module.webservices.rest.web.v1_0.controller.BaseRestController;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.http.HttpStatus;
+import org.springframework.stereotype.Controller;
+import org.springframework.web.bind.annotation.ExceptionHandler;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@Controller
+@RequestMapping(value = "/rest/" + RestConstants.VERSION_1 + "/patient-migration")
+public class PatientsMigrationController extends BaseRestController {
+
+	@Qualifier("patientService")
+	@Autowired
+	private PatientService patientService;
+
+	@RequestMapping(method = RequestMethod.POST)
+	@ResponseBody
+	@ResponseStatus(value = HttpStatus.OK)
+	public void migratePatientLocation(@RequestBody Map<String, Object> body){
+		List<String> patientList = (List<String>) body.get("patientList");
```

Comment:
What happens if patients list isn't an array? - Make sure about the exceptions if possible

Reply:
@VANKINEENITAWRUN

Reply:
![patientListException](https://user-images.githubusercontent.com/30470229/63439743-41ae2080-c44c-11e9-9fbc-619605b32171.png)
@suthagar23  Fixed.